packages/client/modules/meeting/components/EditableTemplatePromptColor.tsx

```diff
@@ -78,7 +80,7 @@ const EditableTemplatePromptColor = (props: Props) => {
   const {groupColor} = prompt
   const {menuProps, menuPortal, originRef, togglePortal} = useMenu<HTMLButtonElement>(
     MenuPosition.UPPER_LEFT,
-    {parentId: 'templateModal'}
+    {parentId: parentId}
```

Comment:
+1 are there any use cases where `parentId` is undefined? if yes, will it work correctly? I see previously it was always defined as `templateModal` now, it's nullable

Reply:
Yes, when the component is not rendered in a modal, `parentId` will be `undefined`. Previously, `parentId` was always `templateModal`, which caused an error when this was rendered outside of a modal.